src/compiler/_normalize-3d-spec.ts

```diff
@@ -18,13 +18,23 @@ export function _fixTrackToWalkaround(t: Track) {
             chr,
             coord
         };
-        const dataTransform: JoinTransform = {
+        const dataTransform: Omit<JoinTransform, 'to'> = {
             type: 'join',
-            from: { url, chromosomeField: chr, genomicField: coord },
-            to: { startField: 'start', endField: 'end' }
+            from: { url, chromosomeField: chr, genomicField: coord }
         };
+        // TODO: need to support other data types as well
+        let to: JoinTransform['to'];
+        if (t.data?.type === 'bigwig') {
+            to = { startField: 'start', endField: 'end' };
```

Comment:
bigwig data comes with `start` and `end` data fields. We can just use that fields for join.